dotnet/src/Functions/Functions.OpenAPI/RestApiOperationRunner.cs

```diff
@@ -29,6 +29,17 @@ internal sealed class RestApiOperationRunner
     /// </summary>
     private readonly Dictionary<string, HttpContentFactory> _payloadFactoryByMediaType;
 
+    /// <summary>
+    /// A dictionary containing the content type as the key and the corresponding content serializer as the value.
+    /// </summary>
+    private static readonly Dictionary<string, HttpResponseContentSerializer> s_serializerByContentType = new()
+    {
+        { "image", async (content) => await content.ReadAsByteArrayAndTranslateExceptionAsync().ConfigureAwait(false) },
+        { "text", async (content) => await content.ReadAsStringWithExceptionMappingAsync().ConfigureAwait(false) },
+        { "application/json", async (content) => await content.ReadAsStringWithExceptionMappingAsync().ConfigureAwait(false)},
+        { "application/xml", async (content) => await content.ReadAsStringWithExceptionMappingAsync().ConfigureAwait(false)}
```

Comment:
nit: Looks like there is no serialization difference between `application/json` and `application/xml`. There is also logic which fetches primary content type `image`, `text`, `application` and tries to get serializer by this type. Wondering, if we can leave just `application` primary content type in this dictionary.

Reply:
Yeah, I initially considered registering a single serialization method using only the primary content-type 'application'. However, I later changed my mind and decided to register only the known ones for now, as there are many different 'application' subtypes for which it is not clear how to serialize - https://www.iana.org/assignments/media-types/media-types.xhtml#application."

Reply:
Makes sense, thank you!